utam-compiler/src/main/java/utam/compiler/grammar/UtamElement.java

```diff
@@ -293,10 +293,6 @@ private Basic() {
       LocatorCodeGeneration locatorHelper = selector.getCodeGenerationHelper(context);
       MethodParametersTracker allParameters = new MethodParametersTracker(String.format("element '%s' getter", name));
       MethodParametersTracker parametersWithoutFilter = new MethodParametersTracker(String.format("element '%s' getter", name));
-      if(scopeElement != null) {
```

Comment:
This block is redundant. The constructor for ElementContext will add the scope element parameters into the parameter list for the target element, if there are any.